src/ert/run_models/everest_run_model.py

```diff
@@ -131,160 +145,7 @@ def __call__(
 
```

Comment:
Is the `MonitorThreadErrorCallback` protocol still needed?

Reply:
No just leftovers I forgot to remove.